odl_video/test_settings.py

```diff
@@ -0,0 +1,5 @@
+"""
+Django test settings for odl_video project.
+"""
+from odl_video.settings import *  # pylint: disable=wildcard-import,unused-wildcard-import
+USWITCH_URL = 'https://testing_odl.mit.edu'
```

Comment:
Instead of setting the URL here can we set it only in the test where we need it set? The `settings` fixture should let you modify settings on a per test basis